receiver/awscontainerinsightreceiver/internal/neuron/neuron_empty_metric_decorator_test.go

```diff
@@ -0,0 +1,287 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package neuron
+
+import (
+	"context"
+	"testing"
+
+	"github.com/open-telemetry/opentelemetry-collector-contrib/receiver/awscontainerinsightreceiver/internal/prometheusscraper/decoratorconsumer"
+	"go.opentelemetry.io/collector/consumer/consumertest"
+	"go.opentelemetry.io/collector/pdata/pmetric"
+	"go.uber.org/zap"
+)
+
+func TestConsumeMetricsForNeuronEmptyMetricsDecorator(t *testing.T) {
```

Comment:
Are you checking that the datapoint value is 0?

Reply:
I will update it